api/app/tests/Unit/Entity/DeputyTest.php

```diff
@@ -22,16 +22,16 @@ public function testGetCourtOrdersWithStatus()
         $courtOrder
             ->setCourtOrderUid($fakeUid)
             ->setType('hybrid')
-            ->setActive(true);
+            ->setStatus('ACTIVE');
```

Comment:
I suggest we introduce some Enums to store the possible statuses of a court order.